platform.md

```diff
@@ -446,6 +454,122 @@ Usage:
+| `<analyzed>`     | `CNB_ANALYZED_PATH`       | `<layers>/analyzed.toml`                          | Path to analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml) |
+| `<app>`          | `CNB_APP_DIR`             | `/workspace`                                      | Path to application directory                                        |
+| `<base-image>`   |                           | resolved from `<analyzed>` when `<kind>` is `run` | Base image to extend                                                 |
+| `<cache-image>`  | `CNB_CACHE_IMAGE`         |                                                   | Reference to a cache image in an OCI image registry                  |
+| `<extender>`     | `CNB_EXTENDER_PATH`       | `<work-dir>/extender.toml`                        | Path to a config file (see [`extender.toml`](#extend-config-toml)    |
+| `<genpkgs>`      | `CNB_GENPKS_PATH`         | `<work-dir>/genpkgs`                              | Path to a `genkpgs` binary                                           |
+| `<ignore-paths>` | `CNB_EXTEND_IGNORE_PATHS` |                                                   | Comma separated list of paths to ignore                              |
+| `<kind>`         |                           | `build`                                           | Type of base image to extend (valid values: `build`, `run`)          |
+| `<log-level>`    | `CNB_LOG_LEVEL`           | `info`                                            | Log Level                                                            |
+| `<target-image>` |                           |                                                   | Target image reference                                               |
+| `<work-dir>`     | `CNB_EXTEND_WORK_DIR`     | `<layers>`                                        | Path to a working directory                                          |
+
+- `<base-image>`, `<cache-image>`, and `<target-image>` MUST be valid image references
+- If after extending the image, the `User` defined on the image is `root`, the lifecycle MUST fail
+- If after extending the run image, the underlying base image has not changed but the `User` defined on the image has changed from its original value, the lifecycle SHOULD warn
+
+When extending the run image:
+- After all image extensions have been applied, the lifecycle MUST set the label `io.buildpacks.rebasable` to `true` if and only if the original base image had `io.buildpacks.rebasable=true` and all image extensions set the label `io.buildpacks.rebasable` to `true`.
+- After all image extensions have been applied, the lifecycle MUST run `<genpkgs>` (if provided) in the context of the extended run image to generate an SBOM for the extended run image. The lifecycle MUST include the SBOM as a layer in the extended run image. The diffID of the layer containing the run image SBOM SHOULD replace the `io.buildpacks.base.sbom` label on the extended run image.
```

Comment:
Basing this off of https://github.com/buildpacks/rfcs/pull/186 - but there is also https://github.com/buildpacks/rfcs/pull/211, which would also work. The platform would need to specify a location to write the generated SBOM.